src/configPaths.ts

```diff
@@ -25,24 +25,30 @@ export type NonConfigEnv = {
 }
 
 export const configPaths = <T extends NonConfigEnv>(
-  process: T
+  process: T,
+  subPath?: string
 ): ConfigTypes => {
   const baseDIR = baseConfigPath(process)
```

Comment:
`baseConfigPath` should not be dynamic any more because its being passed as an argument to this function.